sniff.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
```

Comment:
This file is temporary. It's a tool for me to work out the procedure. It will get removed in the fullness of time.